app/facades/item_facade.rb

```diff
@@ -0,0 +1,18 @@
+class ItemFacade
+  def self.service
+    service ||= ItemService.new
+  end
+
+  def self.all_items
+    item_data = service.get_all_items
+    items = item_data[:data].map do |item|
+      Item.new(item)
+    end
+  end
+
+  def self.find_item(id)
+    all_items.find do |item|
+      item.id == id.to_i
+    end 
+  end
```

Comment:
I wonder if it would be faster to directly hit a separate endpoint for this (assuming one exists), rather than getting all items and comparing the ids with Ruby. This does get it done though!